### PR TITLE
feat: manage marketplace with miactl

### PR DESCRIPTION
**Pull request by malta895**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensible content has been committed

### CODE REVIEW

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:13]: Remember to point out what happens with Windows

Comment (malta895) [reply docs/marketplace/add_to_marketplace/manage_marketplace_items.md:13]: I would not do it here, I would do it in miactl docs, along with the info of the other OSes. I'll open a PR shortly on miactl repo

Comment (malta895) [reply docs/marketplace/add_to_marketplace/manage_marketplace_items.md:13]: Here it is mia-platform/miactl#105

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:24]:
```suggestion
You need to have *Company Owner* or *Project Administrator* role to perform this action
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:60]:
```suggestion
You need to have *Company Owner* or *Project Administrator* role to perform this action
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:69]:
```suggestion
where `ITEM_ID` is an alphanumerical id of the Marketplace element. If you don't know the item id, use the `miactl marketplace list` command to list all the Marketplace Items. You can easily locate the one of interest by looking for its name.
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:77]:
```suggestion
Follow the steps in the [Modifying the Marketplace Item](#enabling-the-visibility-to-all-companies); once you have finished, save the file and apply it to the Marketplace:
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:96]:
```suggestion
You need either the *Company Owner* or *Project Administrator* role at Company level to perform this action
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:20]: I would put here below a brief introduction with a business use case (e.g. ACME Company decides to create an item on the Marketplace: why? --> link to another doc page. how? --> provide here below a practical example with Miactl)

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:45]: We are now mentioning API Portal, but it was not mentioned above. Let's try to make a comprehensive and coherent use case narrative

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:57]:
```suggestion
More information on this point can be found in the [dedicated doc](/cli/miactl/30_commands.md#apply).

### Update an item

```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:56]: Why ACME Company should update it? Let's provide a brief example also in this case to keep the narration going

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:92]: Same as update

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:29]: We may put an explicative screenshot of the "myMarketplaceItem.json" file

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:53]: We can provide a picture of the item correctly created inside the Marketplace of the Console

File: docs/marketplace/add_to_marketplace/contributing_overview.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:34]:
```suggestion
To create Marketplace items you need to be a *Company Owner* or a *Project Administrator* at Company level
```

File: docs/marketplace/add_to_marketplace/contributing_overview.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:40]:
```suggestion
When creating a new Marketplace item, you always need to set the `tenantId` related to the Company you are creating the item in.
```

File: docs/marketplace/add_to_marketplace/contributing_overview.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:67]: which request?

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:31]:
```suggestion
called "My Awesome Service", and you want it to be available in the Marketplace of your Company.
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:34]: i would substitute the json file name with the new name

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:157]:
```suggestion
where `ITEM_ID` is an alphanumerical id of the Marketplace item. If you don't know the item id, use the `miactl marketplace list` command to list all the Marketplace Items. You can easily locate the one of interest by looking for its name.
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:161]:
```suggestion
It is suggested to always download the Marketplace item just before updating it to make sure it works on the latest version.
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:191]:
```suggestion
Imagine you notice that the service "My Awesome Service" is no longer useful for your Company and so you want to delete it.
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:34]: I would talk about the current directory below, when we introduce the command

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:36]:
```suggestion
The file content will probably look like this:
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:132]:
```suggestion
From now on, the service "My Awesome Service" will be visible as a clickable card inside the Marketplace section of the Console.
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:157]:
```suggestion
where `ITEM_ID` is an alphanumerical id of the Marketplace item.  
If you don't know the item id, use the `miactl marketplace list` command to list all the Marketplace Items. You can easily locate the one of interest by looking for its name.
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:165]:
```suggestion
Edit your file following the steps described in the [Modifying the Marketplace Item](#enabling-the-visibility-to-all-companies); 
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:53]: I would remove the reference to API Portal

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:101]:
```suggestion
Notice that the `image` and `supportedByImage` objects are populated with local paths to images: make sure the images exist and that their path is correct.
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:111]:
```suggestion
A message will confirm the operation, returning some information as shown here below:
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:120]:
```suggestion
After the upload, the image keys will be replaced with the `imageUrl` and the `supportedByImageUrl`; to obtain the updated version of the item, use the `get` command: 
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:96]: shouldn't be something like "acme-company" ?

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:92]: Mia-Platform or Acme Company?

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (pmarchini) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:114]:
```suggestion
  ID                  NAME                 STATUS   
```

File: docs/marketplace/add_to_marketplace/manage_marketplace_items.md
Comment (marcofilippi) [inline docs/marketplace/add_to_marketplace/manage_marketplace_items.md:207]:
```suggestion
If you want, you can recreate the item on the Marketplace again by applying the file.

## Open an issue on Mia-Platform Github community page

```

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (pmarchini) [inline docs/marketplace/add_to_marketplace/declarative_marketplace.md:30]: might be better something like `relative file manifests` ?

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (pmarchini) [inline docs/marketplace/add_to_marketplace/declarative_marketplace.md:95]: Do we need to specify the category or explain how to use it?

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (pmarchini) [inline docs/marketplace/add_to_marketplace/declarative_marketplace.md:133]: I would avoid, in the example, restricted metadata fields

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (pmarchini) [inline docs/marketplace/add_to_marketplace/declarative_marketplace.md:226]: After a maintainer merges the changes, a new pipeline is triggered, leading to an update of the item.

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (pmarchini) [inline docs/marketplace/add_to_marketplace/declarative_marketplace.md:236]: Console users can create and add this plugin to their Console projects.

File: docs/marketplace/add_to_marketplace/img/inventory_system.png
Comment (pmarchini) [inline docs/marketplace/add_to_marketplace/img/inventory_system.png:1]: It might be better to avoid using copyrighted materials (acme-corp picture)

File: docs/marketplace/overview_marketplace.md
Comment (pmarchini) [inline docs/marketplace/overview_marketplace.md:55]: I think 'subject to' is better in this case

Comment (malta895) [reply docs/marketplace/overview_marketplace.md:55]:
```suggestion
* **Beta**: implementation is complete, these components won't be subject to any undocumented breaking change but may still include bugs (hence they must be used with caution)
```

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (malta895) [reply docs/marketplace/add_to_marketplace/declarative_marketplace.md:95]: I am in doubt about it. Given the current restriction I don't know what to explain, maybe we put the category `extracategory`?

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (malta895) [inline docs/marketplace/add_to_marketplace/declarative_marketplace.md:133]:
```suggestion
    "tenantId" : "acme-company"
```

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (malta895) [reply docs/marketplace/add_to_marketplace/declarative_marketplace.md:226]:
```suggestion
After a maintainer merges the changes, a new pipeline is triggered, leading to an update of the item.
```

File: docs/marketplace/add_to_marketplace/img/inventory_system.png
Comment (malta895) [reply docs/marketplace/add_to_marketplace/img/inventory_system.png:1]: I took it from wikipedia, it should not be copyrighted https://commons.wikimedia.org/wiki/File:Acme-corp.png

File: docs/marketplace/add_to_marketplace/declarative_marketplace.md
Comment (pmarchini) [reply docs/marketplace/add_to_marketplace/declarative_marketplace.md:95]: I agree with extracategory for the moment

Comment (malta895) [reply docs/marketplace/add_to_marketplace/declarative_marketplace.md:95]:
```suggestion
    "categoryId": "extracategory",
```

**Review by pmarchini — APPROVED** [review]

**Pull request merged**
